peyvandak/Heidari/url_shortener_final/client.py

```diff
@@ -0,0 +1,113 @@
+import requests
+import random
+import numpy as np
+import threading
+import concurrent.futures
+import time
+import traceback
+import json
+
+def login(username, password, timeout=None):
+    data = {'username': username, 'password': password}
+    response = requests.post('http://localhost:8000/users/login', data=data, timeout=timeout)
+    return response
+
+def logout(timeout=None):
+    response = requests.get('http://localhost:8000/users/logout', timeout=timeout)
+    return response
+
+def register(username, password1, password2, timeout=None):
+    data = {'username': username, 'password1': password1, 'password2': password2}
```

Comment:
Why there are 2 passwords in the data?

Reply:
> 
I think it's for conforming the first password